tests/pr_test.go

```diff
@@ -54,11 +71,32 @@ func TestRunAdvancedExample(t *testing.T) {
 func TestRunUpgradeExample(t *testing.T) {
 	t.Parallel()
 
-	options := setupOptions(t, "appid-upg", advancedExampleDir)
+	options := setupOptions(t, "appid-fs-upg", fscloudExampleDir)
 
 	output, err := options.RunTestUpgrade()
 	if !options.UpgradeTestSkipped {
 		assert.Nil(t, err, "This should not have errored")
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunFSCloudSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: fscloudSolutionsDir,
+		Region:       validRegions[rand.Intn(len(validRegions))],
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"ibmcloud_api_key":           options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"],
+		"kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+		"existing_kms_instance_guid": permanentResources["hpcs_south"],
+		"prefix":                     "appid-sol",
```

Comment:
if you pass a value for Prefix in the initial options (line 86), you can then use `options.Prefix` here so it will use a randomly generated prefix too